README.md

```diff
@@ -0,0 +1,61 @@
+# Mosaic 
+
+> :warning: **This package in under early development and not ready for production use, yet.**
+
+`mosaic` provides the functionality to visualize unstructured mesh data on it's native grid within `matplotlib`. 
+Currently `mosaic` only supports MPAS meshes, but future work will add support for other unstructured meshes used in `E3SM`.
+
+## (Developer) Installation 
+
+Assuming you have a working `conda` installation, you can install the latest development version of `mosaic` by running: 
+```
+conda env create --file dev-environment.yml
+conda activate mosaic-dev
+
+python -m pip install -e .
+```
+
+If you have an existing `conda` environment you'd like install the development version of `mosaic` in, you can run: 
+```
+conda install --file dev-environment.yml
```

Comment:
Unfortunately, this doesn't work with the yaml syntax for defining dependencies, only with the text-file based approach (like in MPAS-Analysis).  This is the chief reason I use the text-based approach rather than the yaml approach.  If you switch to the text-based approach (which I recommend), you will need to change the instructions here to:
```suggestion
conda install --file dev-environment.txt
```
and above to:
```
conda create -n mosaic-dev --file dev-environment.txt
```
(i.e. no `env`).

Then, your `dev-environment.txt` will look something like:
```
python >= 3.9
cartopy
cmocean
matplotlib-base
numpy
pyproj
xarray
```